encoding/thrift/register.go

```diff
@@ -79,6 +79,10 @@ type Service struct {
 	Name          string
 	Methods       map[string]UnaryHandler
 	OnewayMethods map[string]OnewayHandler
+
+	// Snippet of Go code representing the function definition of the different
+	// handlers. This is useful for introspection.
+	Signatures map[string]string
```

Comment:
After discussion with @abhinav , conclusion is: let's refactor this to use a single map of procname to struct {} isntead.